packages/native/types/index.d.ts

```diff
@@ -49,60 +47,9 @@ type HostClassComponent<
   { ref?: React.Ref<InstanceType<C>> },
   ReactNativeStyleType<React.ComponentProps<C>>
 >
-type HostFunctionComponent<
-  C extends React.FunctionComponent<any>
-> = CreateStyledComponent<
-  React.ComponentProps<C> & { theme?: Theme },
-  {},
-  {},
-  ReactNativeStyleType<React.ComponentProps<C>>
->
 
-export interface StyledComponents {
-  ActivityIndicator: HostClassComponent<ReactNative['ActivityIndicator']>
-  Button: HostClassComponent<ReactNative['Button']>
-  DatePickerIOS: HostClassComponent<ReactNative['DatePickerIOS']>
-  DrawerLayoutAndroid: HostClassComponent<ReactNative['DrawerLayoutAndroid']>
-  FlatList: HostClassComponent<ReactNative['FlatList']>
-  Image: HostClassComponent<ReactNative['Image']>
-  ImageBackground: HostClassComponent<ReactNative['ImageBackground']>
-  KeyboardAvoidingView: HostClassComponent<ReactNative['KeyboardAvoidingView']>
-  ListView: HostClassComponent<ReactNative['ListView']>
-  Modal: HostClassComponent<ReactNative['Modal']>
-  NavigatorIOS: HostClassComponent<ReactNative['NavigatorIOS']>
-  Picker: HostClassComponent<ReactNative['Picker']>
-  PickerIOS: HostClassComponent<ReactNative['PickerIOS']>
-  Pressable: HostFunctionComponent<ReactNative['Pressable']>
```

Comment:
This was using the HostFunctionComponent - u assume now that all components are typed as classes, this probably wont work correctly